organization/affiliates.md

```diff
@@ -0,0 +1,15 @@
+---
+type: resource
+---
+
+# Affiliates
+
+The Foundation for Public Code is a member-owned association that exists in an ecosystem of affiliates and [partners](partnerships.md). Our affiliates help us to [achieve our mission](https://about.publiccode.net/organization/mission.html). Read more about [becoming an affiliate](../activities/creating-affiliations/index.md).
```

Comment:
Perhaps rephrase "Our affiliates help us to achieve our mission" with "Our affiliates compliment our mission"?

Reply:
I've made an alternate suggestion below that I'm going to update to this.